packages/ui-components/src/Scrollbar/index.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import { Scrollbars } from 'react-custom-scrollbars-2';
+
+export default class CustomScrollbar extends Component {
+
+    constructor(props, ...rest) {
+        super(props, ...rest);
+        this.state = { top: 0, left: 0 };
+        this.handleUpdate = this.handleUpdate.bind(this);
+        this.renderView = this.renderView.bind(this);
+        this.renderThumb = this.renderThumb.bind(this);
+        this.handleWheel = this.handleWheel.bind(this);
+        this.scrollSelf = React.createRef()
+    }
+
+    handleUpdate(values) {
+    }
+
+    renderView({ style, ...props }) {
+        const viewStyle = {};
```

Comment:
there is any merge or other options of `viewStyle`, what's the usage of empty obj.